user_tools/src/spark_rapids_pytools/cloud_api/databricks_aws.py

```diff
@@ -237,11 +237,15 @@ def _init_nodes(self):
                 raise RuntimeError('Failed to find master node information from cluster properties')
             master_nodes_from_conf = {'node_id': None}
         # construct worker nodes info when cluster is inactive
-        if worker_nodes_from_conf is None:
-            worker_node_type_id = self.props.get_value('node_type_id')
-            if worker_node_type_id is None:
-                raise RuntimeError('Failed to find worker node information from cluster properties')
-            worker_nodes_from_conf = [{'node_id': None} for i in range(num_workers)]
+        executors_cnt = len(worker_nodes_from_conf) if worker_nodes_from_conf else 0
+        if num_workers != executors_cnt:
+            self.logger.warning('Cluster configuration: `executors` count %d does not match the '
+                                '`num_workers` value %d. Using generated names.', executors_cnt,
```

Comment:
nit: it may be a bit difficult to understand "Using generated names" without context?

Reply:
Thanks @cindyyuanjiang.  This refers to using default templates as node configurations. I will update the message.